pkgs/development/libraries/libgudev/default.nix

```diff
@@ -25,6 +25,14 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-YSZqsa/J1z28YKiyr3PpnS/f9H2ZVE0IV2Dk+mZ7XdE=";
   };
 
+  patches = lib.optional stdenv.hostPlatform.isMusl
+    ./libgudev-disable-test-double.patch # disable a tests that requires a correct implementation of locales
+  ;
+
+  postPatch = ''
+    substituteInPlace tests/meson.build --replace "LD_PRELOAD=libumockdev-preload.so.0" "LD_PRELOAD=${lib.getLib (umockdev.__spliced.buildHost or umockdev)}/lib/libumockdev-preload.so.0"
```

Comment:
I don't think we need the spliced version of this — we use the unspliced version in nativeCheckInputs anyway.